lib/content-services/src/lib/content-node-share/content-node-share.dialog.ts

```diff
@@ -147,15 +151,15 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
     onToggleExpirationDate(slideToggle: MatSlideToggleChange) {
         if (slideToggle.checked) {
             this.time.enable();
+            this.isExpiryDateToggleChecked = true;
         } else {
             this.time.disable();
             this.time.setValue(null);
+            this.deleteSharedLink(this.sharedId, true);
         }
     }
 
     onDatetimepickerClosed() {
-        this.dateTimePickerInput.nativeElement.blur();
```

Comment:
Why did you remove the blur after datetime picker is closed?

Reply:
added blur after datetimepicker